openc3.sh

```diff
@@ -3,7 +3,7 @@
 set +e
 
 export DOCKER_COMPOSE_COMMAND="docker compose"
-${DOCKER_COMPOSE_COMMAND} version
+${DOCKER_COMPOSE_COMMAND} version &> /dev/null
```

Comment:
Should we pipe the .bat version to NUL?

Reply:
We don't do this on Windows because we assume that we always have "docker compose"